@xen-orchestra/web/src/components/vm/dashboard/VmDashboardRamUsageChart.vue

```diff
@@ -76,7 +76,11 @@ const maxValue = computed(() => {
   return Math.max(...data.stats.memory, 0)
 })
 
-const byteFormatter = (value: number) => {
+const byteFormatter = (value: number | null) => {
+  if (value === null) {
+    return ''
+  }
+
   const result = formatSizeRaw(value, 1)
 
   return `${result?.value}${result?.prefix}`
```

Comment:
add space to match with VmDashboardQuickInfo.vue:36
```suggestion
  return `${result?.value} ${result?.prefix}`
```